pkg/volume/iscsi/iscsi_util.go

```diff
@@ -205,6 +205,18 @@ func (util *ISCSIUtil) AttachDisk(b iscsiDiskMounter) error {
 	iscsiTransport = extractTransportname(string(out))
 
 	bkpPortal := b.Portals
+
+	// create new iface and copy parameters from pre-configured iface to the created iface
+	if b.InitiatorName != "" {
+		// new iface name is <target portal>:<volume name>
+		newIface := bkpPortal[0] + ":" + b.volName
+		err = cloneIface(b, newIface)
```

Comment:
@liggitt 
Thank you for the comment. I think we should decide the SPEC.
There are two options here,
- iSCSI plugin returns error if cloneIface() fails.
- iSCSI plugin continue to create volume using "default" iface.

Currently, the behavior is latter case. But at least, we should log returned error even if the plugin continue to the volume creation.